### PR TITLE
Feature/tabs hashes

### PR DESCRIPTION
**Pull request by axeloz**
Source files for https://github.com/REBELinBLUE/deployer/pull/390

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const mix = require('laravel-mix');
 
 const packages = 'node_modules';
-const build = 'build';
+const build = '../deployer/public';
```
Comment (REBELinBLUE) [inline webpack.mix.js]: please can you revert this line

**Review by REBELinBLUE — CHANGES_REQUESTED** [review]

File: webpack.mix.js (hunk 1 above)
Comment (axeloz) [reply webpack.mix.js]: Ooops sorry, my bad. That allows me to build directly into the deployer document root using `yr run watch-poll`. Fixed

**Comment by REBELinBLUE** [comment]
Can you rebase with master

**Comment by axeloz** [comment]
I did a rebase after a sync between my fork and your repo. I hope I did it well...

**Pull request closed without merging**